### PR TITLE
Update windows crate to 0.35.0

### PR DESCRIPTION
**Pull request by arnabanimesh**

### CODE REVIEW

File: src/window.rs [hunk 1]
```diff
@@ -82,7 +82,6 @@ impl Window {
                 instance,
                 result.as_mut() as *mut _ as _,
             )
-            .ok()?
```
Comment (robmikh) [inline src/window.rs:85]: Could you leave the error handling in place?

**Review by robmikh — CHANGES_REQUESTED** [review]

File: src/window.rs [hunk 2]
```diff
@@ -31,7 +31,7 @@ pub struct Window {
 
 impl Window {
     pub fn new(title: &str, width: u32, height: u32, game: Minesweeper) -> Result<Box<Self>> {
-        let instance = unsafe { GetModuleHandleW(None).ok()? };
```
Comment (robmikh) [inline src/window.rs:34]: Same here.

**Review by robmikh — CHANGES_REQUESTED** [review]

**Comment by robmikh** [comment]
Oh strange, I see you did that now because of compilation errors... I'll accept this and figure out what can be done about that.

**Review by robmikh — APPROVED** [review]

**Comment by kennykerr** [comment]
`HINSTANCE` was affected by a win32 metadata bug https://github.com/microsoft/win32metadata/issues/863

**Comment by kennykerr** [comment]
[0.36.0](https://github.com/microsoft/windows-rs/releases/tag/0.36.0) has been released and includes this fix.

**Pull request merged**